src/site/filters/liquid.js

```diff
@@ -206,6 +206,14 @@ module.exports = function registerFilters() {
     return _.slice(arr, startIndex);
   };
 
+  liquid.filters.formatSharableLinkID = (id, description) => {
+    if (!description) return id;
```

Comment:
I wonder if we should be returning `id` here, or if we should just return `null` and basically not support the element ID at all if `description` isn't defined?

Reply:
hmm, I'm not sure about this one. I think defaulting to an ID would be the safest thing to do and still support anchor linking